### ISSUE TITLE
Cannot properly read tensorflow speech recognition model

### ISSUE THREAD

**Issue by decoybitch** [migueldeicaza/TensorFlowSharp#434]
**Describe the bug**
I am trying to use tensorflow audio recognition model in unity using TensorFlowSharp but getting error when trying to fetch the values.

**To Reproduce**
Download model from http://download.tensorflow.org/models/speech_commands_v0.02.zip

Note: I do not have time to create models, train models, create samples from scratch and then adding your 3-4 lines of code to a sample to debug your problem.   You need to provide a complete test case.

**Expected behaviour**
I manually converted a simple .wav file saying "yes" into tensors and then pasted them into the file. After that, I tried getting the result from the model but I am getting the error:

NullReferenceException: Object reference not set to an instance of an object
TensorFlow.TFSession+Runner.ParseOutput (System.String operation) (at <1fe2de69842a4a4ba15256b83cca05f3>:0)
TensorFlow.TFSession+Runner.AddInput (System.String input, TensorFlow.TFTensor value) (at <1fe2de69842a4a4ba15256b83cca05f3>:0)
NewBehaviourScript.Start () (at Assets/Scripts/NewBehaviourScript.cs:1771)

**Screenshots**
Code:

            string WAV_INPUT = "wav_data";
            string SOFTMAX_NAME = "labels_softmax";
            string[] outputScoresNames = new string[] { SOFTMAX_NAME };

            string[] labels = new string[] { "_silence_", "_unknown_", "stop", "go" };

       

            TextAsset model = Resources.Load("conv_actions_frozen") as TextAsset;
            TFGraph graph = new TFGraph();
            graph.Import(model.bytes);

            TFSession session = new TFSession(graph);

            var runner = session.GetRunner();
        //Debug.Log(typeof(graph[WAV_INPUT][0]));
        //runner.AddInput()
        Debug.Log(graph[WAV_INPUT][0].GetType());
           Line 1771 - runner.AddInput(graph[WAV_INPUT][0].ToString(), tensor);
        Debug.Log("fetching done");
            runner.Fetch(outputScoresNames);
            runner.Run();

**Desktop (please complete the following information):**
 - OS: Mac OS
